### PR TITLE
feat(docs): Add MkDocs skeleton and Getting Started instructions.

### PR DESCRIPTION
**Pull request by lizroth**
*Issue #, if available:*

*Description of changes:*
* Set up MkDocs skeleton
    * https://github.com/aws-samples/aws-security-workshops is our new look n' feel, which uses MkDocs
* First cut Getting Started instructions

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: instructions/docs/getting-started.md [hunk 1]
```diff
@@ -0,0 +1,49 @@
+# Getting Started
+
+## Background
+
+In this section, you will prepare your AWS Environment to work with the Busy Engineer's Document Bucket. You will:
```
Comment (jellywish) [inline instructions/docs/getting-started.md]: What is the aim of learning these things? e.g. "Gain familiarity with the core concepts of encryption using a key provider, and become familiar with encrypting and decrypting across multiple regions"

File: instructions/docs/getting-started.md [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+    * The `BusyEngineersDocumentBucket` stack, with your DynamoDB table and S3 bucket
+    * One CMK in one region, called `Faythe`, that you will use in encryption and decryption operations
+    * One CMK in another region, called `Walter`, that you will use in encryption and decryption operations
+1. Bootstrap the development environment in Cloud9
+1. Start the workshop!
+
+## Make the Change
+
+1. Sign in to your AWS Account for the workshop
+    * Make sure this is **not** a production account!
```
Comment (jellywish) [inline instructions/docs/getting-started.md]: why not?

File: instructions/docs/getting-started.md [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+1. Click **Create Stack** to launch the stack.
+1. [Open the Cloud9 Console](https://us-east-2.console.aws.amazon.com/cloud9/home?region=us-east-2#) to find your Cloud9 IDE.
+1. Click **Open IDE** to launch Cloud9
+    * The repository will check out to `~/environment/workshop` automatically
+1. `cd ~/environment/workshop`
+1. Execute `make bootstrap` to set up your workshop environment
+
+## Try it Out
+
+* `make bootstrap` will take a while. Thank you for your patience while it sets up the environment for you.
```
Comment (jellywish) [inline instructions/docs/getting-started.md]: can you be more specific about how much time it took for you and what it's setting up?

File: instructions/docs/getting-started.md [hunk 4]
```diff
@@ -0,0 +1,49 @@
...
+    * If you are in a live workshop, this is good to run during the presentation.
+    * If you are working on your own, grab a cup of your favorite beverage while you wait.
+* After `make bootstrap` completes, you will have:
+    1. Deployed your workshop stacks using CDK
+    1. Language environments for all of the workshop languages
+* Choose your language of choice for the workshop, and `cd` to its folder under `exercises`
+    * `cd ~/environment/workshop/exercises/node-javascript` for NodeJS
+    * `cd ~/environment/workshop/exercises/python` for Python
+    * `cd ~/environment/workshop/exercises/java` for Java
+* You're ready to get started!
```
Comment (jellywish) [inline instructions/docs/getting-started.md]: I think by bootstrapping, I've already started on this project. But maybe language is a convention (i.e. "getting started" means I've set up my environment)

File: instructions/docs/getting-started.md (hunk 1 above)
Comment (lizroth) [reply instructions/docs/getting-started.md]: Good callout. Will add this expansion.

File: instructions/docs/getting-started.md (hunk 2 above)
Comment (lizroth) [reply instructions/docs/getting-started.md]: Hm, let me try to strike a better balance of not-alarmism and attempts at good advice.

File: instructions/docs/getting-started.md (hunk 4 above)
Comment (lizroth) [reply instructions/docs/getting-started.md]: Good point, tweaked and hopefully improved

File: instructions/docs/getting-started.md (hunk 3 above)
Comment (lizroth) [reply instructions/docs/getting-started.md]: Ah shoot missed this. Will fix

**Pull request merged**